### DISCUSSION TITLE
Consider loading tiles and sprites from text files

### DISCUSSION THREAD

**Discussion by leventp** [paulscottrobson/neo6502-firmware#184] [Implemented Ideas]
MMBasic for Colour Maximite and others has an option to load sprites from text files. One advantage of this is that Sprite Editor programs written in Basic can generate these files and can be loaded via TILE or SPRITE commands (or IMAGE). It is also possible to generate these files manually as the entries are quite visual.  The implementation could be totally different,  just wanted to give some ideas from other Basic implementations. 

The format could be something like this (based on MMBasic):

t | s, quantity, rows, columns   (where t is for tile and s is for sprite definition)
{rows * cols pixel definitions}

pixel represented by colour numbers (in case of NeoBasic 0 to F). Space or 0 is used for transparency. Guess another option to have pixel colour values comma separated so more than 16 colours can be defined for future releases.

For example : One 16 by 16 sprite definition (which is a green filled square):

s,1,16,16
2222222222222222
2222222222222222
2222222222222222 
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222
2222222222222222

**Comment by paulscottrobson** [comment]
Something like this would work fine, gload could do this automatically.
